Fix the following error, due to "node:fs/promises" not being available in
jammy's version of nodejs:

Error: Cannot find module 'node:fs/promises'
Require stack:
- .../build.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (.../build.js:1:40)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '.../build.js' ]
}

(Note: nyc/lib/fs-promises.js is from the node-istanbul package)

--- a/build.js
+++ b/build.js
@@ -1,4 +1,4 @@
-const { readFile, mkdir, writeFile } = require('node:fs/promises');
+const { readFile, mkdir, writeFile } = require('nyc/lib/fs-promises.js');
 const { minify } = require('terser');
 
 (async function() {
